apps/web/app/(app)/environments/[environmentId]/people/[personId]/page.tsx

```diff
@@ -1,10 +1,94 @@
-import PersonDetails from "./PersonDetails";
+import { getPersonWithAttributeClasses } from "@formbricks/lib/services/person";
+import { getResponsesByPersonId } from "@formbricks/lib/services/response";
+import { getSessionWithActionsOfPerson } from "@formbricks/lib/services/session";
+import { getDisplaysOfPerson } from "@formbricks/lib/services/displays";
+import { getSurvey } from "@formbricks/lib/services/survey";
+import AttributesSection from "@/app/(app)/environments/[environmentId]/people/[personId]/AttributesSection";
+import ActivitySection from "@/app/(app)/environments/[environmentId]/people/[personId]/ActivitySection";
+import HeadingSection from "@/app/(app)/environments/[environmentId]/people/[personId]/HeadingSection";
+import ResponseSection from "@/app/(app)/environments/[environmentId]/people/[personId]/ResponseSection";
+import { TResponseWithSurveyData } from "@formbricks/types/v1/responses";
+
+export default async function PersonPage({ params }) {
+  let [personWithAttributes, displays, sessionsWithActions, responses] = await Promise.all([
```

Comment:
you don't need to fetch all these in page.tsx. You can also fetch them in the server-component you need it. Next.js takes care of caching and deduping.